packages/code.rb

```diff
@@ -65,5 +65,8 @@ def self.postinstall
     puts 'Now, please run \'code-oss\' to start Code.'.lightgreen
     puts 'Happy coding!'.lightgreen
     puts
+    puts 'If you\'re having issues with your installation, comment out the Sommelier'.orange
+    puts ' configuration in `~/.bashrc` and try again.'
```

Comment:
This is not a good suggestion because it will break other apps that run fine with sommelier.

Reply:
Also, you could update to https://github.com/Microsoft/vscode/archive/1.31.1.tar.gz.

Reply:
Yeah we should add in some language about how this will break other sommelier apps, but for now it is the only way we can get Code to run right now.

Reply:
Does `stopsommelier` do the job?

Reply:
I'm tired of how the `sommelier`, `broadway`, and `xdg_base` packages work right now. I'm rewriting them.

Reply:
@JL2210 Can you put some language in there explaining that this might break other sommelier apps and then I'll merge it?